### PR TITLE
support .ts, .json and .yaml config

### PR DESCRIPTION
**Pull request by guoyunhe**
### Why am I submitting this PR

support .ts, .json and .yaml config

### Does it fix an existing ticket?

No

### Checklist

- [x] only relevant code is changed (make a diff before you submit the PR)
- [x] tests are included and pass: `yarn test` (see [details here](https://github.com/i18next/i18next-parser/blob/master/docs/development.md#tests))
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by guoyunhe** [comment]
Here is one unit test failed. But I believe it is not related to my changes.

File: src/helpers.js [hunk 1]
```diff
@@ -281,6 +286,37 @@ function makeDefaultSort(pluralSeparator) {
   }
 }
 
+async function esConfigLoader(filepath) {
+  return (await import(filepath)).default
+}
+
+async function tsConfigLoader(filepath) {
+  console.log(filepath)
```
Comment (karellm) [inline src/helpers.js:294]: `Console.log` :)

**Review by karellm — COMMENTED** [review]
Thanks for the PR. Can you remove the console.log ?

File: src/helpers.js (hunk 1 above)
Comment (guoyunhe) [reply src/helpers.js]: done

**Comment by karellm** [comment]
Published as `7.0.0`

**Pull request merged**